pywbem/cim_operations.py

```diff
@@ -1172,12 +1237,302 @@ def __repr__(self):
                "default_namespace=%r, x509=%s, verify_callback=%r, " \
                "ca_certs=%r, no_verification=%r, timeout=%r, " \
                "use_pull_operations=%r, stats_enabled=%r, recorders=%s)" % \
-               (self.__class__.__name__, self.url, creds_repr, self.conn_id,
-                self.default_namespace, x509_repr, self.verify_callback,
-                self.ca_certs, self.no_verification, self.timeout,
-                self.use_pull_operations, self.stats_enabled,
+               (self.__class__.__name__, self.url, creds_repr,
+                self.conn_id, self.default_namespace, x509_repr,
+                self.verify_callback, self.ca_certs, self.no_verification,
+                self.timeout, self.use_pull_operations, self.stats_enabled,
                 recorder_list)
 
+    @classmethod
+    def configure_api_logger(cls, log_dest=DEFAULT_LOG_DESTINATION,
+                             detail_level=DEFAULT_API_LOG_DEFAULT_LEVEL,
+                             log_filename=DEFAULT_LOG_DESTINATION,
+                             connection=None):
+        """
+        A log configuration helper function to configure a logger  logs
+        execution/response/exceptions of the WBEMConnection methods that
+        communicate with a WBEM server.
```

Comment:
A "that" is missing between "logger" and "logs".
We should use the same terminology here as we used in the logging section: "... logs user-issued calls to :class:`~pywbem:WBEMConnection` methods that drive WBEM operations (see :ref:`WBEM operations`) and the responses before they are passed back to the user."

Reply:
Done